src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/validate/PNCIdentifier.kt

```diff
@@ -5,11 +5,21 @@ import java.time.LocalDate
 
 const val LONG_PNC_ID_LENGTH = 10
 
-class PNCIdentifier(inputPncId: String? = null) {
+interface PNCIdentifier {
+
+  val pncId: String
+  companion object {
+    fun create(inputPncId: String? = null) = ValidPNCIdentifier(inputPncId)
```

Comment:
will soon do the validation in here and create the correct type of PNCIdentifier